requirements/dev-mac-requirements.in

```diff
@@ -1,4 +1,4 @@
--r dev-requirements.in
+-r dev-sdw-requirements.in
```

Comment:
Shouldn't this be `dev-bullseye-requirements.in` (the dev requirements that match the production Python version)?

Reply:
oops, you're right